src/components/Card/Card.component.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Card.styles.css'
+
+function Card({element, hideList,item}){
+    React.useState(()=>{
+
+        console.log('CARD COMPONENTE'); 
+        console.log(item); 
+
+    },[])
+
+
+    return (    
+        <div className = "card" onClick = {()=>hideList(item)}>
+            <img src={element.thumbnails.high.url} alt=""  width = "300"/>
+            <h3>{element.title}</h3>            
+        </div>
+    )
```

Comment:
is a best practice to assign a value to the `alt` attr for `img` element because of user experience and accessibility. 

on this case, you could assign the `element.title` value :D